Libs/MRML/Core/vtkMRMLDisplayNode.h

```diff
@@ -252,17 +278,29 @@ class VTK_MRML_EXPORT vtkMRMLDisplayNode : public vtkMRMLNode
   vtkBooleanMacro(Clipping, int);
 
   /// Set the slice intersection visibility of the display node.
-  /// \sa SliceIntersectionVisibility, GetSliceIntersectionVisibility(),
+  /// Function to manage \sa Visibility2D for backwards compatibility
```

Comment:
You could use `\deprecated`

Reply:
It's actually there already, just on the bottom of the dox, right above the function definition.